doc/tutorials/introduction/test_writing/accuracy_test_writing.markdown

```diff
@@ -0,0 +1,269 @@
+# Writing Accuracy Tests
+
+# {#tutorial_accuracy_test_writing}
+
+`This testing framework is based on Google Tests`
+
+There are two major types of C++ tests: *accuracy/regression* tests and *performance* tests.  Each module can have two test binaries: `opencv_test_<MODULE_NAME>` and `opencv_perf_<MODULE_NAME>`, and two tests folders: `<opencv(_contrib)/modules/<MODULE_NAME>/test>` and `<opencv(_contrib)/modules/<MODULE_NAME>/perf>`. These applications can be built for every supported platform (Win/Lin/Mac/Android).
+
+## ACCURACY TESTS
+
+### Work directory
+
+All modules have their own dir for accuracy tests: `opencv_contrib/modules/(moduleName)/test/...`
```

Comment:
> dir

use full name

---

> opencv_contrib

contrib only?

---

> All modules have

This is not true. Some contrib modules are live without any tests.
Statement should be rephrased.

Reply:
done